6.2_run2000.py

```diff
@@ -0,0 +1,22 @@
+import time
+
+
+def timer(f, *args, **kwargs):
```

Comment:
Replace 'f' Variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing

Reply:
I changed f to func